tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/amqp/QueueAutoCreationTest.java

```diff
@@ -124,7 +124,7 @@ public void testAutoCreateOnTopic() throws Exception {
       Connection connection = factory.createConnection();
       SimpleString addressName = UUIDGenerator.getInstance().generateSimpleStringUUID();
       logger.debug("Address is {}", addressName);
-      clientSession.createAddress(addressName, RoutingType.ANYCAST, false);
+      clientSession.createAddress(addressName, RoutingType.MULTICAST, false);
```

Comment:
I went ahead and just backed this out entirely. The original test passes with my changes.

Reply:
That I dont quite understand. The same change was and still is being made in the basically-same test in  [AutoCreateJmsDestinationTest.java](https://github.com/apache/activemq-artemis/pull/4421/files#diff-6afa2ca2a98c6d517b31d95108be8a9a803968254c6f27bb739bc0bed395c2ae), so now we have 2 tests creating addresses with different routing types, one multicast and one anycast, and yet both sending to them as a Topic...isnt that exactly 'sending msgs to address w/mismatching routing types' ?

Reply:
You're right that is strange. Both tests succeed because the address with the wrong routing-type is automatically updated. I updated it so both tests use the correct routing-type to being with.